dotnet/flatpak-dotnet-generator.py

```diff
@@ -17,6 +17,10 @@ def main():
     parser.add_argument('output', help='The output JSON sources file')
     parser.add_argument('project', help='The project file')
     parser.add_argument('--runtime', '-r', help='The target runtime to restore packages for')
+    parser.add_argument('--freedesktop', '-f', help='The target version of the freedesktop sdk to use', 
+                        choices=['21.08', '22.08'], default='21.08')
```

Comment:
Any reason this defaults to `21.08` vs `22.08`? (Might be better to drop the choices too, so this doesn't need updates every runtime release.)

Reply:
I chose to keep it at 21.08 to mirror how the script works atm, but may as well bump it to 22.08. Agreed on dropping the choices, at the time it made sense to make it near impossible to get a failure due to an incorrect runtime version.

Reply:
fixed as of d68dceb0c3686bed8fc75f59ed48ae73ad6dfc2d.